docs/source/topics/proc_accessing-the-internal-openshift-registry.adoc

```diff
@@ -75,7 +75,7 @@ $ oc get is
 +
 [subs="+quotes,attributes"]
 ----
-$ oc set image-lookup
+$ oc set image-lookup --all
```

Comment:
Wouldn't it be better to restrict this to `oc set image-lookup alpine` rather than `--all`, in case the user has other image streams where short name lookups would not be desirable?

Reply:
@cfergeau yes sure, we can just use the image name which user want to add for lookup.